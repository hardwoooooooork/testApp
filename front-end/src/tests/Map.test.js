import React from 'react';
import { render } from '@testing-library/react';
import Map from '../components/Maps';


describe('<Map />', () => {
    it('matches snapshot', () => {
        const utils = render(<Map />);
        expect(utils.container).toMatchSnapshot();
    });
});