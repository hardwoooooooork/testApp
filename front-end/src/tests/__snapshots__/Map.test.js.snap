// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map /> matches snapshot 1`] = `
<div>
  <div>
    <div
      class="ui container"
      id="Maps"
    >
      <div
        class="title"
      >
        <h2
          class="ui header"
        >
          Map List
        </h2>
        <button
          class="ui button"
        >
          Logout
        </button>
      </div>
      <div
        class="ui grid container"
      >
        <div
          class="four wide column map-find-area"
        >
          <div
            class="ui vertical map-find-menu menu"
          >
            <div
              class="item find-input"
            >
              <form>
                <div
                  class="ui icon input"
                  placehold="Search ....."
                >
                  <input
                    required=""
                    type="text"
                    value=""
                  />
                  <i
                    aria-hidden="true"
                    class="search circular inverted link icon"
                  />
                </div>
              </form>
            </div>
            <div
              class="item find-result-area"
            >
              <div
                class="ui list"
                role="list"
              >
                <div
                  class="item"
                  role="listitem"
                >
                  no data
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="twelve wide column map-view-area"
        >
          <div>
            Looooooooooooading....
          </div>
        </div>
        <div
          class="hot-keyword-area"
        >
          <div
            class="ui vertical hot-keyword menu"
            style="display: none;"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
